app/Http/Controllers/ReportsController.php

```
@@ -419,23 +354,181 @@ function splitAmountInTwo($amount) {
+                    'status' => JournalEntry::STATUS_POSTED,
+                    'journal_details_account_no' => $account->account_number,
+                    'journal_details_ref_no' => $lastSeries,
+
+                ];
+
+                if ($subsidiaryCategory->sub_cat_code === SubsidiaryCategory::INSUR_ADD) {
+                    if ($account->pivot->transaction_type == 'credit') {
+                        $details['journal_details_credit'] = 0;
+                    } else {
+                        $details['journal_details_debit'] =  0;
+                    }
```

PHPMD (2.15.0)

The method postDepreciation uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)